src/spritekit.cs

```diff
@@ -2372,5 +2558,372 @@ interface SKReferenceNode {
 		[Export ("resolveReferenceNode")]
 		void Resolve ();
 	}
+
+	[iOS (9,0)][Mac (10,11)]
```

Comment:
Oh yes I forgot to mention that, introspection and monotouch-tests ran on iOS 9.3.